README.markdown

```diff
@@ -17,7 +18,31 @@ Tech this uses [thnx]:
 
 ---
 
-#### Setup
+## Create a new theme
+
+```
+git clone git@github.com:interglobalvision/globie-wordpress-starter-pack.git /path/to/new-theme
+cd /path/to/new-theme
+rm -rf .git
+git init
+```
+
+## Dev
+
+```
+git clone git@github.com:interglobalvision/globie-wordpress-starter-pack.git /path/to/new-theme
+cd /path/to/new-theme
+```
+
+Locally exclude built files and stuff. Edit `.git/info/exclude`:
+
+```
+composer.lock
+vendor
+dist/
+```
+
```

Comment:
Thats only for developing the `globie-wordpress-starter-pack` not for making new themes